airflow/contrib/operators/mlengine_operator.py

```diff
@@ -387,6 +400,9 @@ class MLEngineVersionOperator(BaseOperator):
         For this to work, the service account making the request must have
         domain-wide delegation enabled.
     :type delegate_to: str
+
```

Comment:
Trim this newline please

Reply:
ok but, I tried to keep the same blank line as in other parameters, I did not want to trim all these existing blank lines. should I?